build.sbt

```diff
@@ -53,7 +54,7 @@ lazy val jawnSettings = Seq(
   Test / publishArtifact := false,
   Compile / doc / sources := {
```

Comment:
We don't need this setting anymore because Dottydoc works now. Together with #348 this means all the `build.sbt` changes should become unnecessary.

Reply:
Look like now one able to merge this PR. Anyway, in #348 `isDotty` will be removed. Or, if #348 will be merged first, I'll pull the changes in this PR.